src/RichTextEditor.js

```diff
@@ -392,3 +393,7 @@ export {
   Button,
   Dropdown,
 };
+
+export default RichTextEditor = React.forwardRef((props, ref) => (
+  <RichTextEditorBase forwardedRef={ref} {...props} />
+));
```

Comment:
This is definitely a breaking change; anyone whose code was previously passing `forwardedRef` will stop working if we land this.

Reply:
Also, I'm confused about the `RichTextEditor =` part here, that seems like a syntax error.